org.civicrm.reqangular/src/common/angular-date/src/decorators/DatepickerDirectiveDecorator.js

```diff
@@ -0,0 +1,24 @@
+function DatepickerDirectiveDecorator($delegate) {
```

Comment:
*(wherever else applies)*
Please provide a description about what this decorator does